mk/defs.mk

```diff
@@ -4,17 +4,23 @@ BUILD_DEBUG ?= 0
 # Build AVX backend or not
 BUILD_AVX ?= 0
 
+# Build SSE backend or not
+BUILD_SSE ?= 0
+
+# Build CPU backend or not
+BUILD_CPU ?= 0
```

Comment:
The generic/portable C backend should be regarded as the fallback. It should not be required to be selected.

Reply:
It has been removed.